src/brainCloudClient-authentication.js

```diff
@@ -108,6 +108,7 @@ function BCAuthentication() {
 			bc.authentication.AUTHENTICATION_TYPE_ANONYMOUS,
 			null,
 			forceCreate,
+            null,
```

Comment:
Sorry for formating, Every new code I use 4 spaces. The whole source is a mismatch of both. I don't want to fix it all because it will screw with gitblame and history